dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -175,6 +177,7 @@ public class WorkflowResource extends AbstractWorkflowResource<Workflow>
     private static final Logger LOG = LoggerFactory.getLogger(WorkflowResource.class);
     private static final String PAGINATION_LIMIT = "100";
     private static final long MAX_PAGINATION_LIMIT = 100;
+    private static final long MAX_LIMIT = 200;
```

Comment:
Could you just reuse the max pagination limit of 100 (constant above)? I assume the only reason we returned 200 is because there was no pagination so we tried to return as much as possible